pkg/pipelines/statustracker/deployment.go

```diff
@@ -0,0 +1,153 @@
+package statustracker
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/odo/pkg/pipelines/meta"
+	"github.com/openshift/odo/pkg/pipelines/roles"
+)
+
+const (
+	operatorName   = "commit-status-tracker"
+	containerImage = "quay.io/kmcdermo/status-tracker"
+)
```

Comment:
Got 403 Unauthorized when I tried tdo pull this image.   Please make sure it is readable to everyone.

Reply:
It is, but we should probably put it under the quay.io/redhat-developer organisation.

Reply:
I would, but I can't create new repositories in the redhat-developer org, @sbose78 can you help?

Reply:
Made `kmcdermo` an owner in redhat-developer 👍 .

Reply:
@bigkevmcd  Please provide the source repo of the image.

Reply:
https://github.com/tektoncd/experimental/tree/master/commit-status-tracker